src/test/resources/org/jabref/logic/importer/pdfContentImporterTest-pdfs.bib

```diff
@@ -0,0 +1,110 @@
+@inproceedings{se2paper,
```

Comment:
- The bib file should be in THE SAME directory as the .pdf files
- Set `.` as library-specific directory (in the library properties)
- Have a `file` attached to each entry in the `bib` file.
- Try to improve BibTeX data in the `bib` file:
  -  Add DOI
  - Fix authors
  - Add Year
  - Generate citation key
  - Rename PDF files to nicer name.